### DISCUSSION TITLE
1.2.3 Milestone Discussion

### DISCUSSION THREAD

**Discussion by bacoords** [holger1411/understrap#2120] [General]
I've set up a new milestone for [1.2.3](https://github.com/understrap/understrap/milestone/12) that would be a quicker release than some of the bigger (potentially breaking) changes that would be better suited for 1.3. Here's what I'm thinking so far, but open to discussion:

- [x] https://github.com/understrap/understrap/pull/2073
- [x] https://github.com/understrap/understrap/pull/2010
- [x] https://github.com/understrap/understrap/pull/2071
- [x] https://github.com/understrap/understrap/pull/2118
- [x] https://github.com/understrap/understrap/pull/2035
- [x] https://github.com/understrap/understrap/pull/2091
- [x] https://github.com/understrap/understrap/pull/2121
- [x] https://github.com/understrap/understrap/pull/2032
- [x] https://github.com/understrap/understrap/pull/1967
- [x] https://github.com/understrap/understrap/pull/2113
- [x]  https://github.com/understrap/understrap/pull/2130 
- [x] https://github.com/understrap/understrap/pull/2129 
- [x] https://github.com/understrap/understrap/pull/2127

Other possibilities

- [ ] https://github.com/understrap/understrap/issues/2107
- [ ] ??

CC @IanDelMar

**Comment by IanDelMar** [comment]
#1967 - just throws a notice. But we need to change the version number from 1.3.0 to 1.2.3 if it is merged for the v1.2.3 release.
#2035 - important to merge asap
#2113 - does not introduce potentially breaking changes
#2118 - does not introduce potentially breaking changes

**Reply by bacoords** [reply]
I've updated my list above. 
#2035 and #2118 are merged.
Merging #2113 opens up a new issue: #2123
We should be fine to merge #1967 I just want to do a little more testing on it

**Reply by IanDelMar** [reply]
https://github.com/understrap/understrap/issues/2123 is addressed in https://github.com/understrap/understrap/pull/2113

**Comment by bacoords** [comment]
@IanDelMar What are your thoughts about committing the rebuilt CSS/JS here - https://github.com/understrap/understrap/blob/cf16154caa254c934b2faa2cef4471f05bbe49c2/.github/workflows/build.yml#L68C14-L68C38 

What's the reason for committing the changed files versus just testing that the build is working properly but not committing any changes?

**Reply by IanDelMar** [reply]
Well, we want to know what changes, if any, are caused by changes to either scss files or dependencies. When you commit the changes, you have a diff and can see those changes immediately. I find auto-committing things very handy. For example: Because of the auto-commit, I noticed that we need to bump the understrap postcss plugin in package-lock.json because there was this unexpected change:
![image](https://github.com/understrap/understrap/assets/42134098/e47cba75-39a4-4e36-b013-522bb75f3b61)
Maybe there's another way to see these changes like a do diff and post the result in a comment, but most of the time you want to commit these changes anyway.

There is still a workflow issue. I need to check what to do about pushes that fail because of branch protection. When I tested the workflow in my repositories I didn't check for this scenario...

What are your thoughts?

**Reply by bacoords** [reply]
Yeah that makes sense. I'm not sure there's anything we can do about the branch protection issue

**Reply by bacoords** [reply]
I also noticed that the Linting action isn't running on PRs

**Reply by bacoords** [reply]
I take it back, we just haven't been changing any PHP files on the last few PRs, so I'm guessing it doesn't need to run.

**Reply by IanDelMar** [reply]
Yep, the linting workflow isn't supposed to run on PRs that do not change PHP files.

**Comment by IanDelMar** [comment]
@bacoords You have listed https://github.com/understrap/understrap/issues/2107 under "Other possibilities". I don't know what to do about this issue. I can't reproduce the error.

**Reply by bacoords** [reply]
Agreed - I don't think there's anything to do on that one for this release.
